macros/CSSRef.ejs

```diff
@@ -189,7 +189,7 @@ if (slug) {
     }
 
     function buildSublist(pages, title, noCodeTags) {
-      var result = '<li data-default-state="open"><a href="#"><strong>' + title + '</strong></a><ol>';
+      var result = '<li class="toggle"><details open><summary>' + title + '</summary><ol>';
```

Comment:
Here's the difference for ``{CSSRef}`` on the page https://developer.mozilla.org/en-US/docs/Web/CSS/transform-function (👍 correctly converted):

<img width="731" alt="pr 789 cssref" src="https://user-images.githubusercontent.com/286017/45325771-7a910600-b517-11e8-9b03-d97128431282.png">

The whitespace is slightly different, and the bold text for the title, but these are not significant changes.